gfs/chunkserver/chunkserver.go

```diff
@@ -31,3 +31,23 @@ func InitChunkServer(csAddr int) {
 
 	// TO:DO Constantly check for the end of any leases the chunkserver has. How to prevent TOCTOU bugs here?
 }
+
+func InitBenchmarkingChunkServer(csAddr int, config services.BenchmarkConfig) {
+	lis, err := net.Listen("tcp", ":"+strconv.Itoa(csAddr))
+	if err != nil {
+		log.Fatalf("failed to listen: %v", err)
+	}
+
+	s := services.NewBenchmarkingChunkServer(strconv.Itoa(csAddr), config)
+	grpcServer := grpc.NewServer()
+	protos.RegisterChunkServerServer(grpcServer, &s)
+
+	// Start Chunkserver
+	go func() {
+		if err := grpcServer.Serve(lis); err != nil {
+			log.Fatalf("failed to serve: %s", err)
+		}
+	}()
+
+	// TO:DO Constantly check for the end of any leases the chunkserver has. How to prevent TOCTOU bugs here?
+}
```

Comment:
Unlike how I implemented `InitBenchmarkingClient()`, where I made wrapper function around `InitBenchmarkingClient()`, I copy-pasted `InitChunkServer()` to here because there was an issue of returning a copy-by-value of the Mutex in the chunkServer.